etna/search/views.py

```diff
@@ -284,6 +293,28 @@ def get_meta_title(self) -> str:
             title += ' for "' + query.replace('"', "'") + '"'
         return title
 
+    def get_gtm_content_group(self) -> str:
+        return "Search"
+
+    def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
+        data = super().get_datalayer_data(request)
+        data["customDimension3"] = self.__class__.__name__
+        try:
+            data["customDimension8"] = (
+                self.title_base + ": " + self.form.cleaned_data["group"]
+            )
```

Comment:
Could we keep dict updates consistent usage [example](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/records/models.py#L399):  for changes in this PR

`data.update(customDimension8=self.title_base + ": " + self.form.cleaned_data["group"])`